README.md

```diff
@@ -4,30 +4,36 @@ Makelangelo Software is a Java program that prepares art for CNC plotters.  It i
 
 It pairs really well with [Marlin 3D printer firmware](https://github.com/MarginallyClever/Marlin-polargraph/commits/polargraph-2.0.x), the code in the brain of the robot that receives instructions and moves the motors.  
 
-![screenshot](screenshot.png)
+![screenshot](screenshot.jpg)
 
 ![workflow](https://github.com/MarginallyClever/Makelangelo-software/actions/workflows/maven.yml/badge.svg)
 
 ## Where to go from here
 
-**On your own**  [Reading our friendly manuals](http://mcr.dozuki.com).  Pictures and arrows and everything.  
+**On your own**  [Reading our friendly manuals](http://mcr.dozuki.com). Pictures and arrows and everything.  
 
 **Together** and [join our Discord group](https://discord.gg/QtvHqAv8yp)
 
 # Start!
 
-- Setup [Marlin 3D printer firmware](https://github.com/MarginallyClever/Marlin-polargraph/commits/polargraph-2.0.x) with the help of the [documentation](https://www.marginallyclever.com/2021/10/friday-facts-4-how-to-marlin-polargraph/)
-- Install "Makelangelo Software" on your computer to control the robot 
-  1. Install The latest OpenJDK (java software)
-  2. Grab a [nightly build](https://github.com/MarginallyClever/Makelangelo-software/releases/tag/Nightly) and run it directly *or* Install Makelangelo Software App from the source
+- Buy a plotter [Makelangelo 5](https://www.marginallyclever.com/products/makelangelo-5/) or build your own;
+  ![plotter](src/main/resources/splash.png)
+- Setup [Marlin 3D printer firmware](https://github.com/MarginallyClever/Marlin-polargraph/commits/polargraph-2.0.x) with the help of the [documentation](https://www.marginallyclever.com/2021/10/friday-facts-4-how-to-marlin-polargraph/);
```

Comment:
Maybe the tree https://github.com/MarginallyClever/Marlin-polargraph/tree/polargraph-2.0.x or directly the .zip https://github.com/MarginallyClever/Marlin-polargraph/archive/refs/heads/polargraph-2.0.x.zip
and not the commit page ?

Reply:
thanks, corrected